src/harness/testcases/WINNF_FT_S_EXZ_testcase.py

```diff
@@ -0,0 +1,315 @@
+    grant_0 = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_0['operationParam']['operationFrequencyRange'] = {
+        'lowFrequency': 3625000000.0,
+        'highFrequency': 3635000000.0
+    }
+
+    # 2. With frequency range partially overlapping with the frequency 
+    #     range of the nearest exclusion zone.
+    grant_1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_1['operationParam']['operationFrequencyRange'][
+        'lowFrequency'] = 3625000000.0
+    grant_1['operationParam']['operationFrequencyRange'][
+        'highFrequency'] = 3635000000.0
+
+    # 3. With frequency range partially overlapping with the frequency 
+    #     range of the nearest exclusion zone.
+    grant_2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_2['operationParam']['operationFrequencyRange'][
+        'lowFrequency'] = 3625000000.0
+    grant_2['operationParam']['operationFrequencyRange'][
+        'highFrequency'] = 3635000000.0
+
+    # Create the actual config.
+    devices = [device_a, device_b, device_c]
+    grants = [grant_0, grant_1, grant_2]
+    config = {
+        'fccIds': [(d['fccId'], 47) for d in devices],
+        'userIds': [d['userId'] for d in devices],
+        'registrationRequests': devices,
+        'expectedResponseCodes': [(0,), (0,), (0,)],
+        'grantRequests' : grants,
+        'expectedGrantResponseCodes': [(0,), (400,), (400,)],
+    }
+    writeConfig(filename, config)
+
+
+
+  @configurable_testcase(generate_EXZ_1_default_config)
+  def test_WINNF_FT_S_EXZ_1(self,config_filename):
+    """Injected Exclusion Zones
+
+    The response should be SUCCESS for CBSDs located outside 50meters of all 
+    Exclusion Zones.
+    The responseCode = 400 for CBSDs located within 50meters of all Exclusion Zones 
+    or inside Exclusion Zones. 
+    """
+
+    #Load Exclusion Zones
+    exz_record = json.load(
+        open(os.path.join('testcases', 'testdata', 'exz_record_0.json')))
+
+    # Inject EXZ database records
+    # TODO Need to figure out how to send freqRange
+    """
+    zone_id = self._sas_admin.InjectExZone({'record':exz_record}, \
+                     {'frequencyRange':{'lowFrequency': 3620000000.0,\
+                      'highFrequency': 3630000000.0}})
+    """
+    self._sas_admin.InjectExZone({'record': exz_record})
+
+    config = loadConfig(config_filename)
+    # Very light checking of the config file.
+    self.assertEqual(
+        len(config['registrationRequests']),
+        len(config['expectedResponseCodes']))
+
+    # Inject FCC IDs.
+    for fcc_id in config['fccIds']:
+      self._sas_admin.InjectFccId({'fccId': fcc_id})
+
+    # Inject user IDs.
+    for user_id in config['userIds']:
+      self._sas_admin.InjectUserId({'userId': user_id})
+   
+    # Register N2, N3 and N4 CBSDs.
+    device_cert = self.getCertFilename('dp_client.cert')
```

Comment:
Default is (or should be) to use a DP cert so this step is unnecessary.

Reply:
Removed dp cert creation line